lib/modules/manager/maven/index.ts

```diff
@@ -5,6 +5,9 @@ import * as mavenVersioning from '../../versioning/maven';
 export { extractAllPackageFiles } from './extract';
 export { bumpPackageVersion, updateDependency } from './update';
 
+export const url = 'https://maven.apache.org/index.html';
```

Comment:
```suggestion
export const url = 'https://maven.apache.org';
```
I would drop the index file